backend/modules/simulation/service.py

```
@@ -340,24 +340,31 @@ def save_message(
         scene_id: int,
         sender_name: str,
         message_content: str,
-        message_type: str
+        message_type: str,
+        session_id: str = None
```

Ruff (0.15.6)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)